EKF/estimator_interface.h

```diff
@@ -159,7 +159,7 @@ class EstimatorInterface
 	int getNumberOfActiveHorizontalAidingSources() const;
 
 	// return true if the local position estimate is valid
-	bool local_position_is_valid() const { return !_deadreckon_time_exceeded; }
+	bool local_position_is_valid() const { return local_position_is_valid(); }
```

Comment:
This could be deleted entirely if you also delete the call in print_status().

https://github.com/PX4/PX4-ECL/blob/67f13f7ededd4eb21e10abb6445f3f0fc4b7a6e5/EKF/estimator_interface.cpp#L605

Reply:
Done